migrations/versions/3d3df26524b7_commentset_membership.py

```diff
@@ -0,0 +1,67 @@
+"""commentset membership.
+
+Revision ID: 3d3df26524b7
+Revises: daeb6753652a
+Create Date: 2020-12-04 13:03:31.208857
+
+"""
+
+from alembic import op
+from sqlalchemy.dialects import postgresql
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '3d3df26524b7'
+down_revision = 'ad5013552ec6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'commentset_membership',
+        sa.Column('granted_at', sa.TIMESTAMP(timezone=True), nullable=False),
+        sa.Column('revoked_at', sa.TIMESTAMP(timezone=True), nullable=True),
+        sa.Column('record_type', sa.Integer(), nullable=False),
+        sa.Column('commentset_id', sa.Integer(), nullable=False),
+        sa.Column(
+            'last_seen_at',
+            sa.TIMESTAMP(timezone=True),
+            nullable=False,
+            default=sa.func.utcnow(),
```

Comment:
`default` doesn't work in migrations. Can be safely removed here as the migration doesn't affect existing data.